docs/user-guide/configure.md

```diff
@@ -375,6 +375,35 @@ Patterns are applied against the file path relative to the directory of the conf
 
 Overrides have higher precedence than regular configurations. Multiple overrides within the same config are applied in order. That is, the last override block in a config file always has the highest precedence.
 
+## `processors`
+
+> ⚠️ Warning:
```

Comment:
[suggest] Let's use the special notation supported by GitHub. Ref https://github.com/orgs/community/discussions/16925

```suggestion
> [!WARNING]
```